api/v1alpha1/lvmcluster_types.go

```diff
@@ -17,13 +17,19 @@ limitations under the License.
 package v1alpha1
 
 import (
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+const (
+	// ConditionLVMClusterCRvalid communicates if LVMCluster CR is invalid
+	ConditionLVMClusterCRvalid conditionsv1.ConditionType = "LVMClusterCRvalid"
```

Comment:
can we remove the `CR` from this name?  I think LVMCluster and LVMClusterCR both should imply the same thing.

Reply:
Agreed.

Reply:
done